.github/workflows/release.yml

```diff
@@ -0,0 +1,270 @@
+name: Release (neo-cli)
+
+on:
+  release:
+    types: [published]
+
+env:
+  DOTNET_VERSION: 8.0.x
+  CONFIGURATION: Release
+  DIST_PATH: /tmp/dist
+  OUTPUT_PATH: /tmp/out
+
+jobs:
+  build-leveldb:
+    name: Build leveldb (windows-latest)
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        arch: [x64, arm64]
+
+    steps:
+    - name: Lookup Cache Distribution
+      id: cache-leveldb
+      uses: actions/cache@v4
+      with:
+        path: ./leveldb/build/Release/*
+        key: leveldb-windows-${{ matrix.arch }}
+        enableCrossOsArchive: true
+        lookup-only: true
+
+    - if: ${{ steps.cache-leveldb.outputs.cache-hit != 'true' }}
```

Comment:
comment all these cache and what they are

Reply:
Checks for a cached version of `libleveldb.dll` exists. If doesn't exists than build it